workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreatorFromMetadata.cs

```diff
@@ -67,11 +68,11 @@ public GameObject OnEntityCreated(SpatialOSEntity entity)
             }
 
             var gameObject = Object.Instantiate(prefab, position, Quaternion.identity);
-            gameObject.name = $"{prefab.name}(SpatialOS: {entity.SpatialOSEntityId}, worker {workerType}";
+            gameObject.name = $"{prefab.name}(SpatialOS: {entity.SpatialOSEntityId}, Worker: {workerType}";
```

Comment:
Can we close the open bracket with a ")"? Can we add the entity index back again?

Reply:
The entity index was removed because it really doesn't make sense if this API is supposed to be about abstracting away the ECS and apparently doesn't even identify the Entity uniquely in itself (which is a reason for either exposing nothing or exposing the entire Entity, and the former makes more sense in the context)

Reply:
will add the ")" back in though :)